src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -198,4 +202,10 @@
     <Compile Remove="build\*.cs" />
     <None Include="build\**\*.cs;build\**\*.targets" Pack="true" PackagePath="build" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Xamarin.Forms.Platform.GTK" Version="5.0.0.1791-pre5" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
+    <PackageReference Include="Xamarin.Forms.Platform.GTK" Version="5.0.0.1791-pre5" />
+  </ItemGroup>
```

Comment:
Can you please use existing ItemGroups (such groups already created)

Reply:
Stupid tooling doing this...